home/private_dot_local/lib/agent-equipment/agent_equipment/inventory.py

```
@@ -0,0 +1,1233 @@
+
+def _admission_error(code: str, message: str) -> AdapterError:
+    return AdapterError(
+        code=code,
+        classification="invalid_request",
+        message=message,
+        retry="never",
+        mutation_state="not_started",
+    )
+
+
+def _plain_snapshot_canonical_bytes(
```

Ruff (0.16.1)

[warning] Too many return statements (9 > 6)

(PLR0911)